README.md

```diff
@@ -91,6 +91,8 @@ ActionCable is included to enable the [Turbo Streams](https://turbo.hotwired.dev
 1. Optionally create `manifest.yml` and variable files for cloud.gov deployment
 1. Optionally create Github Actions workflows
 1. Optionally create CircleCI workflows
-1. Optionally configure DAP (Digital Analytics Program)
 1. Optionally create [Architecture Decision Records](https://adr.github.io/) for above setup
+1. Optionally create a New Relic config with FEDRAMP-specific host
```

Comment:
I realized I left off the New Relic documentation, and decided just to reorder to match order they're asked about in `rails new`